lib/rules/at-rule-descriptor-no-unknown/index.mjs

```diff
@@ -0,0 +1,56 @@
+import { lexer } from 'css-tree';
```

Comment:
It looks no type definitions have been added.
See: 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7f6d332bcd59a641e035ac645259603fa6ed0f01/types/css-tree/index.d.ts

I will submit a PR to upstream.

Reply:
If the upstream PR may be a blocker, we can also work around the type error via patch-package, such as <https://github.com/stylelint/stylelint/pull/7543/files#diff-9005bb639a6220616aebaf301054701ddc479adaac5c8885c0825e735794d234>. Once the upstream PR is merged, we can remove the patch and bump the type package safely. 👌🏼

Reply:
Thanks for sharing the steps!
I added in [this commit](https://github.com/stylelint/stylelint/pull/8197/commits/ae422f2fd1a84f75b271f7b84b8344bd83f94c36).